experiments.py

```diff
@@ -3,10 +3,59 @@
 import copy
 
 
+def run_adaptive_SCI(params, clipping_methods):
```

Comment:
This is a question for everyone: for our main set of experiments, shall we just run `run_adaptive_SCI` and `run_adaptive_II`? Since we compare fixed here as well we probably don't need to do more parameter comparisons in the CL evaluation unless folks think its necessary.